From ac7ddf7ed6eae1106080619d432340c0d964acba Mon Sep 17 00:00:00 2001
From: Richard Zhu <hongxing.zhu@nxp.com>
Date: Wed, 31 Oct 2018 16:21:05 +0800
Subject: [PATCH 4961/5242] MLK-20142-1 ARM64: dts: use internal pll as pcie
 ref on 8mm ddr4 board

commit  b14c8e1ca77a0515d0b28703331880befe8a3630 from
https://source.codeaurora.org/external/imx/linux-imx.git

Since the 8mm ddr4 evk board is same to the -revb evk board.
Configure the pcie ref clk as the internal pll clock.

Signed-off-by: Richard Zhu <hongxing.zhu@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../boot/dts/freescale/fsl-imx8mm-ddr4-evk.dts     |   10 ++++++++++
 1 file changed, 10 insertions(+)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8mm-ddr4-evk.dts b/arch/arm64/boot/dts/freescale/fsl-imx8mm-ddr4-evk.dts
index 1e34598..ee54c1c 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8mm-ddr4-evk.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8mm-ddr4-evk.dts
@@ -50,3 +50,13 @@
 &flexspi {
 	status = "disabled";
 };
+
+/*
+ * External OSC is used as PCIe REFCLK on RevC board.
+ * DDR4 board is same to the RevB board, configure
+ * PCIe REFCLK to internal PLL.
+ */
+&pcie0{
+	ext_osc = <0>;
+	status = "okay";
+};
-- 
1.7.9.5

